src/realtps_import/src/jobs.rs

```diff
@@ -38,7 +38,7 @@ impl JobRunner {
             Ok(new_jobs) => new_jobs,
             Err(e) => {
                 print_error(&e);
-                error!("error running job. repeating");
+                error!("error running job: {}. repeating", &e);
```

Comment:
The line above this is also printing the error.

It would be good to be able to grep for "error running job" and see the error on the same line. Maybe instead change `print_error` to accept a string indicating what was happening, like `print_error("running job", &e)`.

There's another use of `print_error` further down this file.